### PR TITLE
feat: Initialize images when submitting My Collection artworks

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Feature**

This PR resolves [CX-2418]

### Description

Initialize images when submitting My Collection artworks with the following approach:

1. Save the image URLs of the MC artwork in `initialPhotos` inside the `ArtworkSubmissionModel`,
2. Add the photos later in `UploadPhotosForm` using the URLs from `initialPhotos`.

https://user-images.githubusercontent.com/4691889/159454748-0838e3df-14d7-4e84-a13b-f2bf059c6635.mp4

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [x] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Initialize images when submitting My Collection artworks - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2418]: https://artsyproduct.atlassian.net/browse/CX-2418?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/utils/calculatePhotoSize.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const totalSizeLimitInBytes = 30000000
 export const calculateSinglePhotoSize = (photo: Photo): Photo => {
   if (!photo.size) {
     photo.error = true
-    photo.sizeDisplayValue = "Size not found"
```
Comment (AlicanAkyuz) [inline src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/utils/calculatePhotoSize.ts:10]: one idea: 

we can check if photo has a `path` attribute, if not we can set an appropriate message, maybe “automatically added” like you suggested. if it has a path, we could then keep this `photo.size` check and keep "Size not found".

File: src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/PhotoRow.tsx [hunk 2]
```diff
@@ -27,17 +26,17 @@ export const PhotoRow: React.FC<Props> = ({ photo, onPhotoDelete }) => {
 }
 
 const PhotoRowContent: React.FC<Props> = ({ photo, onPhotoDelete }) => (
-  <Flex flexDirection="row">
-    <Flex width="53%" justifyContent="center">
+  <Flex flexDirection="row" justifyContent="space-between" width="100%">
+    <Flex alignItems="center">
```
Comment (olerichter00) [inline src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/PhotoRow.tsx:30]: This needed some alignment to not cut off the "Size not found" or "automatically added" text. @AlicanAkyuz can you check whether this change aligns with the design?

File: src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/utils/calculatePhotoSize.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/utils/calculatePhotoSize.ts:10]: I've added a `automaticallyAdded` field to `photos` to indicate that the photos have been automatically added.

**Review by MounirDhahri — APPROVED** [review]
🙌

**Review by rajsam003 — DISMISSED** [review]
👏

**Review by dariakoko — APPROVED** [review]
👍

File: src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/PhotoRow.tsx (hunk 2 above)
Comment (AlicanAkyuz) [reply src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/UploadPhotos/PhotoRow.tsx]: I've checked it and it looks aligned with designs 👍

**Review by AlicanAkyuz — APPROVED** [review]

**Comment by AlicanAkyuz** [comment]
@olerichter00 should I merge this PR? It looks ready to me. 

one side note not related to this PR but the other one where you've fixed the location auto-complete: when I choose a location, the autosuggest box stays (video below).  should we have a look at this in this PR or elsewhere? 

https://user-images.githubusercontent.com/42584148/159689624-b5eb09df-9db2-4044-8cfb-0290fa06059c.mp4

**Comment by AlicanAkyuz** [comment]
@olerichter00 nevermind my comment above about location thingy, I've fixed it here: https://github.com/artsy/eigen/pull/6489

**Comment by olerichter00** [comment]
> @olerichter00 nevermind my comment above about location thingy, I've fixed it here: #6489

Thank you very much 🙇

**Pull request merged**